netlify/netlify-plugin-nextjs
# chore: move cypress out of demo and fix set up

### Summary

- Move cypress out of demo
- Add ci specific config and dev specific config
- Fix testing-library types not being found

This PR is the base in which all the tests are written on top of so should be merged first.

### Test plan

1. Visit the Deploy Preview Logs: https://app.netlify.com/sites/netlify-plugin-nextjs-demo/deploys/6184066e70c9010007ed5ccf#L379
2. See that cypress runs the testing-library test correctly
---
1. Checkout this branch
2. Run `npm run cy:open`
3. In the GUI that pens, run the test
4. See that it runs the test and everything passes

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal (small picture)
https://github.com/netlify/pod-ecosystem-integrations/issues/33

### Standard checks:


---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
